docs/plugins/discovery-ec2.asciidoc

```diff
@@ -122,6 +121,10 @@ Defaults to `private_ip`.
     How long the list of hosts is cached to prevent further requests to the AWS API.
     Defaults to `10s`.
 
+*All* secure settings of this plugin are {ref}/secure-settings.html#reloadable-secure-settings[reloadable].
+After you reload the settings, the hosts provider, during dynamic nodes refresh,
```

Comment:
Maybe `After you reload the settings, an aws sdk client with the latest settings from the keystore will be used`?